### PR TITLE
Type RT reg code in scc wizard

### PR DESCRIPTION
**Pull request by mloviska**
- Needles: <del>[osd](https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/1150)</del> && [Additional Yast2 SCC needles](https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/1152)
- Verification run: [sle-12-SP4-Server-DVD-RT-x86_64-Build0019-sles+rt-net@64bit](http://eris.suse.cz/tests/14686)

### CODE REVIEW

**Comment by asmorodskyi** [comment]
you are in generic function which used by a lot of different scenarios , I think you need more then one verification run . also I suspect you would need to put some condition before it ...

File: lib/registration.pm [hunk 1]
```diff
@@ -214,6 +214,7 @@ sub register_addons {
             else {
                 assert_and_click("scc-code-field-$addon", timeout => 240);
             }
+            send_key 'alt-s';
```
Comment (rwx788) [inline lib/registration.pm]: Actually, this key press should not be needed, as field is selected by `assert_and_click`. So we just need proper needle here. I guess the ones you've created just have too big area as click is performed in the center of it (AFAIK).

Comment (mloviska) [reply lib/registration.pm]: I have modified the needle, and it is still failing to type the reg code

Comment (mloviska) [reply lib/registration.pm]: * http://eris.suse.cz/tests/14632#step/addon_products_via_SCC_yast2/33

It's a bugger :/

Comment (rwx788) [reply lib/registration.pm]: Thing is that name of the field depends on the name of the module, so we should at least use first letter of the `$addon` and then we might remove code above...

Comment (mloviska) [reply lib/registration.pm]:
```
[2019-05-21T12:31:38.732 CEST] [debug] <<< testapi::assert_and_click(mustmatch='scc-code-field-rt', button='timeout', timeout=240)
[2019-05-21T12:31:38.732 CEST] [debug] clicking at 502/495
```

**Comment by mloviska** [comment]
@asmorodskyi maybe we do not need to do more VRs, since I have added RT  particular needles and just updated the way how we call *assert_and_click*. WDYT ?

**Comment by mloviska** [comment]
Last 25 runs
http://eris.suse.cz/tests/14701#next_previous

**Comment by asmorodskyi** [comment]
> @asmorodskyi maybe we do not need to do more VRs, since I have added RT particular needles and just updated the way how we call _assert_and_click_. WDYT ?

agree , LGTM . @rwx788  ok for you ?

**Comment by rwx788** [comment]
With new needle looks like we could get rid of module specific needles =). But that works only in case window is maximized. Let's try it out.

**Pull request merged**